README.md

```diff
@@ -1,47 +1,49 @@
-        - retrieve nonce
-        - submit signed transaction
-        - retrieve tx status
-
-# Filecoin signing library
-
-### Main features:
-
-- CBOR <-> JSON serialization and deserialization
-- Secp256k1 signing, etc.
-- Can be compiled as WASM
-- API specs (work in progress)
-
-# Examples (Work in progress)
-
-- *examples/jsonrpc* minimal JS examples/integration tests for signing service
-- *examples/www* how to use the signing library when compiled as WASM from a browser
-- *examples/wasm* integration tests written in JS for WASM
-
-# References
-
-[Template/Tytorial: How to use a WASM npm package](https://rustwasm.github.io/docs/wasm-pack/tutorials/npm-browser-packages/using-your-library.html)
+  - Focus: Exchange integration
+  - Exposes most of the functions available in the signing library
+  - Lotus integration:
+    - nonce caching
+    - determine testnet vs mainnet
+    - retrieve nonce
+    - submit signed transaction
+    - retrieve tx status
+- Examples
+
+  | From            | To              | Status                           |                                  |
+  | --------------- | --------------- | -------------------------------- | -------------------------------- |
+  | Node.js         | JSONRPC Service | Ready :heavy_check_mark:         | [Link](examples/service_jsonrpc) |
+  |                 |                 |                                  |                                  |
+  | Browser         | WASM            | Ready :heavy_check_mark:         | [Link](examples/wasm_browser)    |
+  | Browser         | WASM + Ledger   | Planned :hourglass_flowing_sand: | [Soon]()                         |
+  | Node.js / Mocha | WASM            | Ready :heavy_check_mark:         | [Link](examples/wasm_node)       |
+  |                 |                 |                                  |                                  |
+  | Rust            | Rust + Ledger   | Planned :hourglass_flowing_sand: | [Soon]()                         |
+  | C               | Rust            | Ready :heavy_check_mark:         | [Link](examples/ffi/c)           |
+  | C++             | Rust            | Ready :heavy_check_mark:         | [Link](examples/ffi/c++)         |
+  | Java            | Rust            | Ready :heavy_check_mark:         | [Link](examples/ffi/java)        |
+  | Kotlin          | Rust            | Ready :heavy_check_mark:         | [Link](examples/ffi/kotlin)      |
+  | Go              | Rust            | Planned :hourglass_flowing_sand: | [Soon]()                         |
+  | Objective-C     | Rust            | Planned :hourglass_flowing_sand: | [Soon]()                         |
+  | Swift           | Rust            | Planned :hourglass_flowing_sand: | [Soon]()                         |
+  | React Native    | Rust            | Planned :hourglass_flowing_sand: | [Soon]()                         |
+  | Flutter         | Rust            | Planned :hourglass_flowing_sand: | [Soon]()                         |
```

Comment:
Nice table!

Looks like the `from (calling) to (callee)` is a convention although I always thought as `from (callee) to (calling)`.

Reply:
I agree that we can replace to/from with caller/callee, we can later reorganize the table when to do a final documentation review. I think there are many things to improve there.